pkg/util/liveconfig/dev.go

```diff
@@ -0,0 +1,37 @@
+package liveconfig
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	HIVEENVVARIABLE = "HIVEKUBECONFIGPATH"
```

Comment:
can we make this a bit more readable?
`HIVE_ENV_VARIABLE = "HIVE_KUBE_CONFIG_PATH"`